src/ci/github-actions/jobs.yml

```diff
@@ -37,7 +37,7 @@ runners:
 
 envs:
   env-x86_64-apple-tests: &env-x86_64-apple-tests
-    SCRIPT: ./x.py --stage 2 test --skip tests/ui --skip tests/rustdoc --skip tests/run-make-fulldeps
+    SCRIPT: ./x.py --stage 2 test --skip tests/ui --skip tests/rustdoc
```

Comment:
This removes test coverage from Apple, in theory. I wonder if we should instead change it to `--skip tests/run-make` and add `tests/run-make` to `x86_64-apple-2`. Although some of the tests were actually moved to `ui-fulldeps` :thinking:

Reply:
From what I can tell, the Apple test suites are split between two jobs: one that skips (ui, rustdoc, run-make-fulldeps), and one that only runs those jobs.

So I don't think this actually affects overall test coverage on Apple?

Reply:
Yes, this was preexisting. When the tests moved from -fulldeps to just run-make, we stopped testing them on Apple.